src/datetimepicker.ios.ts

```diff
@@ -189,7 +189,7 @@ export class DateTimePicker extends DateTimePickerBase {
         }
         if (color) {
             if (this.SUPPORT_TEXT_COLOR) {
-                nativePicker.setValueForKey(color, 'textColor');
+                nativePicker.setValueForKey(color.ios, 'textColor');
```

Comment:
Interesting, so the prior code is failing on I assume NS7?    Does it throw any errors?

Reply:
we are on NS 6.5.20 actually !
The application crashes instantly, but sentry does catch an error for us:
```
[TNSDictionaryAdapter set]: unrecognized selector sent to instance 0x28239bdc0
```

Reply:
It was my fault. I removed ".ios" on my last PR.
I'm sorry.